Tutorials/Processing/unsorted/conditionals/bouncingball/web-export/index.html

```diff
@@ -4,9 +4,9 @@
 		<title>moving_ball : Built with Processing and Processing.js</title>
```

Comment:
what is this crazy file?!

Reply:
I don't know :P It's some processing.js version of a processing sketch that's in the folder above. Strange, but I fixed it rather than deleting it so we can leave it in until you've time to work out whether it should be in here

Reply:
I'm almost sure these files are unnecessary and just there by accident, but they aren't hurting anyone either :smile: